nats/src/main.rs

```diff
@@ -206,6 +207,8 @@ impl NatsMessagingProvider {
                 ));
             }
         };
+
+        // Use the first visible cluster_uri
         let url = cfg.cluster_uris.get(0).unwrap();
```

Comment:
Interesting... I'm not sure why we have a vec if we only ever use the first entry 🤔